android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ListViewProxy.java

```diff
@@ -350,6 +379,69 @@ public void setActivity(Activity activity)
 		}
 	}
 
+	// NOTE: For internal use only.
+	public KrollDict getContentOffset()
+	{
+		final TiListView listView = getListView();
+
+		if (listView != null) {
+			final KrollDict contentOffset = new KrollDict();
+
+			final int x = (int) new TiDimension(listView.getScrollOffsetX(),
+				TiDimension.TYPE_WIDTH, COMPLEX_UNIT_DIP).getAsDefault(listView);
+			final int y = (int) new TiDimension(listView.getScrollOffsetY(),
+				TiDimension.TYPE_HEIGHT, COMPLEX_UNIT_DIP).getAsDefault(listView);
+
+			contentOffset.put(TiC.PROPERTY_X, x);
+			contentOffset.put(TiC.PROPERTY_Y, y);
+
+			// NOTE: Since obtaining the scroll offset from RecyclerView is unreliable
+			// when items are added/removed, also grab the current visible item instead.
+			final int currentIndex = listView.getAdapterIndex(listView.getFirstVisibleItem().index);
```

Comment:
This line causes a crash if the ListView is empty, because `getFirstVisibleItem()` will return `null` in this case. Maybe we should remove the "index" entry from the `contentOffset` dictionary in this case?

You can reproduce this the below. Tapping on the "Editing" button will cause the `getContentOffset()` method to be called. Alternatively, switch between dark/light mode will cause this too.
```javascript
const window = Ti.UI.createWindow();
const listView = Ti.UI.createListView();
const isEditing = () => `EDITING: ${listView.editing ? 'ON' : 'OFF'}`;
const editButton = Ti.UI.createButton({
	title: isEditing(),
	bottom: 5,
	right: 5,
});
editButton.addEventListener('click', () => {
	listView.editing = !listView.editing;
	editButton.title = isEditing();
});
window.add([listView, editButton]);
window.open();
```